src/test/kotlin/racing/CarRacingKoTest.kt

```diff
@@ -1,38 +1,49 @@
 package racing
 
+import io.kotest.assertions.throwables.shouldThrow
 import io.kotest.core.spec.style.DescribeSpec
 import io.kotest.matchers.shouldBe
-import java.io.ByteArrayOutputStream
-import java.io.PrintStream
+import racing.domain.Car
+import racing.domain.CarRacing
+import racing.domain.RacingCars
+import racing.domain.StaticPowerStrategy
+
+private fun testCar(name: String, position: Int = 0) =
```

Comment:
```suggestion
private fun TestCar(name: String, position: Int = 0) =
```
함수 명명규칙에는 맞지 않지만 이렇게 가짜 생성자를 쓰기도 합니다. [list builder](https://kotlinlang.org/docs/constructing-collections.html#initializer-functions-for-lists) 같이요.

Reply:
list builder 처럼 언더바라도 안뜨면 좋을텐데, 눈에 밟히네요 ㅜㅜ

Reply:
워닝이 좀 그렇죠. 내로남불 같은 느낌? ㅋㅋ